omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -13,18 +8,81 @@ Nginx Gunicorn configuration (Unix/Linux)
       see :ref:`python-requirements` on the
       :doc:`/sysadmins/version-requirements` page.
 
-Install `Django 1.8`_ and `Gunicorn <http://docs.gunicorn.org/>`_ using
-package requirements file:
+.. _gunicorn_default_configuration:
+
+Gunicorn default configuration (Unix/Linux)
+-------------------------------------------
+
+Install `Django 1.8`_,
+`Gunicorn >= 19.3 <http://gunicorn.org>`_
+and `futures <https://pypi.python.org/pypi/futures>`_ using package requirements file:
```

Comment:
Missed that thanks. I looked in the snoopy merge for related PRs, but I assume it was omitted for Travis reasons. Can you still explain the requirement though?